app/models/event.rb

```diff
@@ -83,8 +83,9 @@ def self.format_date(type)
 
   #scope
   scope :recent_events, -> {order(created_at: :DESC).limit(12)}
-  scope :featured_events, -> {order(created_at: :DESC).limit(2)}
-  scope :popular_events, -> {order(created_at: :DESC).limit(3)}
+  scope :featured_events, -> {order(created_at: :DESC).limit(12)}
+  scope :popular_events, -> {find_by_sql("Select events.*, COUNT(bookings.event_id) AS num from events INNER JOIN bookings WHERE bookings.event_id = events.id GROUP BY events.id ORDER BY num")}
```

Comment:
Line is too long. [193/80]<br>Space missing inside {.<br>Space missing inside }.